// pose/out/cli.js
#!/usr/bin/env node
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.opts = exports.action = undefined;

var _yargs = require('yargs');

var _yargs2 = _interopRequireDefault(_yargs);

var _path = require('path');

var _path2 = _interopRequireDefault(_path);

var _fs = require('fs');

var _fs2 = _interopRequireDefault(_fs);

var _ = require('.');

var pose = _interopRequireWildcard(_);

var _actions = require('./actions');

var _actions2 = _interopRequireDefault(_actions);

function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj.default = obj; return newObj; } }

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var opts = _yargs2.default.usage('$ pose <action> [options]').help('help').epilog(_actions2.default.trim()).options({
  name: {
    alias: ['n'],
    desc: 'Name of template'
  },

  context: {
    alias: ['c'],
    desc: 'Entry of template'
  },

  help: {
    alias: ['h']
  }
}).argv;
var action = opts._[0];

if (!action) {
  console.error('Type "pose help" to get started.');
  process.exit();
}

[opts._pose = _path2.default.join(process.env.HOME, '.pose'), opts._templates = _path2.default.join(opts._pose, 'templates')].forEach(function (dir) {
  _fs2.default.mkdir(dir, function () {
    return null;
  });
});

if (typeof pose[action] === 'function') {
  pose[action](opts);
} else {
  console.error('Unknown action "' + action + '"');
}
exports.action = action;
exports.opts = opts;
