// pose/out/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.help = exports.rm = exports.remove = exports.ls = exports.list = exports.mk = exports.save = exports.i = exports.init = undefined;

var _init = require('./init');

var _init2 = _interopRequireDefault(_init);

var _help = require('./help');

var _help2 = _interopRequireDefault(_help);

var _save = require('./save');

var _save2 = _interopRequireDefault(_save);

var _list = require('./list');

var _list2 = _interopRequireDefault(_list);

var _remove = require('./remove');

var _remove2 = _interopRequireDefault(_remove);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

// aliases
var i = _init2.default;
var mk = _save2.default;
var ls = _list2.default;
var rm = _remove2.default;

// actions
exports.init = _init2.default;
exports.i = i;
exports.save = _save2.default;
exports.mk = mk;
exports.list = _list2.default;
exports.ls = ls;
exports.remove = _remove2.default;
exports.rm = rm;
exports.help = _help2.default;
